src/rating/Rating.js

```diff
@@ -138,25 +153,57 @@ export default class Rating extends Component {
 
   getCurrentRating() {
     const { value } = this.state;
-    const { imageSize, ratingCount } = this.props;
+    const { fractions, imageSize, ratingCount } = this.props;
     const startingValue = ratingCount / 2;
     let currentRating = 0;
 
     if (value > ratingCount * imageSize / 2) {
       currentRating = ratingCount;
     } else if (value > imageSize) {
-      currentRating = Math.ceil(startingValue + value / imageSize);
+      currentRating = startingValue + value / imageSize;
+      currentRating = !fractions
+        ? Math.ceil(currentRating)
+        : +currentRating.toFixed(fractions);
```

Comment:
Sounds fine to me. Can't think of a use case that needs more than 2 decimal places anyway.